lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -1287,34 +1308,43 @@ class NGPHistoryParser {
     return this;
   }
 
+  buildCalibrationRecords(events) {
+    for (const event of this.eventsOfType(NGPHistoryEvent.EVENT_TYPE.CALIBRATION_COMPLETE)) {
+      // In the Tidepool data model, calibration event timestamps match against the SMBG timestamp.
```

Comment:
I'm not sure if that is always true? For other devices I've just used the timestamp of the calibration event. The reason why they (almost) always match is because most devices don't generate a separate SMBG event, in which case it needs to be created. I guess in this case you want to know when the measurement happened, not when the calibration was accepted, so it makes sense.

Reply:
I was basing it from [this data model documentation](http://developer.tidepool.org/data-model/device-data/types/deviceEvent/calibration.html#subtype).

Reply:
:+1: